webapp/src/user_settings.ts

```diff
@@ -1,41 +1,50 @@
-import Client from 'client';
-import manifest from 'manifest';
+import Client from "client";
+import manifest from "manifest";
 
-export default function getUserSettings(serverRoute: string, disabled: boolean) {
-    return {
-        id: manifest.id,
-        icon: `${serverRoute}/plugins/${manifest.id}/public/icon.svg`,
-        uiName: manifest.name,
-        action: disabled ? {
-            title: 'Connect your Microsoft Teams Account',
-            text: 'Connect your Mattermost and Microsoft Teams accounts to get the ability to link and synchronise channel-based collaboration with Microsoft Teams.',
-            buttonText: 'Connect account',
-            onClick: () => window.open(`${Client.url}/connect?from_preferences=true`),
-        } : undefined, //eslint-disable-line no-undefined
-        sections: [makeNotificationsSetting(disabled)],
-    };
+export default function getUserSettings(
+  serverRoute: string,
+  disabled: boolean
+) {
+  return {
+    id: manifest.id,
+    icon: `${serverRoute}/plugins/${manifest.id}/public/icon.svg`,
+    uiName: manifest.name,
+    action: disabled
+      ? {
+          title: "Connect your Microsoft Teams Account",
+          text: "Connect your Mattermost and Microsoft Teams accounts to get the ability to link and synchronise channel-based collaboration with Microsoft Teams.",
```

Comment:
Should we update this legacy text?

Reply:
@lieut-data @esethna @asaadmahmood - How likely are we to remove the enable notification prompt flow at this point? And when would we make this call by? I ask because I'm finalizing a training video for Microsoft Teams scheduled for release with v10 (Sept 17), and want to ensure the video produced provides an accurate representation of the user's experience.

Reply:
```suggestion
          text: "Connect your Mattermost and Microsoft Teams accounts to receive notifications in Mattermost for chats and group chats when you're away or offline in Microsoft Teams.",
```